layer/layer_data.h

```diff
@@ -15,6 +15,7 @@
 #ifndef STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_LAYER_DATA_H_
 #define STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_LAYER_DATA_H_
 
+#include <chrono>
```

Comment:
same here

Reply:
`layer_data .h` is using `time_point::min()`. Doesn't that count?